pkg/cloud/services/ec2/instances.go

```diff
@@ -108,14 +108,24 @@ func (s *Service) CreateInstance(scope *scope.MachineScope) (*infrav1.Instance,
 		RootDeviceSize: scope.AWSMachine.Spec.RootDeviceSize,
 	}
 
+	additionalTags := infrav1.Tags{}
+	// Merge in AWSCluster additional tags
+	for k, v := range scope.AWSCluster.Spec.AdditionalTags {
+		additionalTags[k] = v
+	}
+	// Merge in AWSMachine additional tags - these take priority over AWSCluster additional tags
+	for k, v := range scope.AWSMachine.Spec.AdditionalTags {
+		additionalTags[k] = v
+	}
```

Comment:
Hit this in GCP yesterday, What do you think about adding another method to tags? https://github.com/kubernetes-sigs/cluster-api-provider-gcp/pull/143/commits/4ce5f143216dfe0743dfebcbaa44219000b9a2af#diff-7e70cd56370673a32d574971f458326cR75-R81

Reply:
+1, maybe a MergeTags method?

Reply:
I'm writing Merge now